### PR TITLE
[13.0][ADD] stock_location_route_description: add a description field

### PR DESCRIPTION
**Pull request by JudithAforgeFlow**
Added a description field on routes

CC @ForgeFlow

### CODE REVIEW

File: stock_location_route_description/models/stock_location_route.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class StockLocationRoute(models.Model):
+    _inherit = "stock.location.route"
+
+    description = fields.Char(string="Description")
```
Comment (AaronHForgeFlow) [inline stock_location_route_description/models/stock_location_route.py]: description = fields.Char()

**Review by AaronHForgeFlow — COMMENTED** [review]
The readme is missing, isn't it?

**Comment by MiquelRForgeFlow** [comment]
Yeah, a readme folder with some fragments is needed :)

File: stock_location_route_description/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import stock_location_route
```
Comment (DavidJForgeFlow) [inline stock_location_route_description/models/__init__.py:1]: Shouldn't it be stock_location_route_description? The rest LGTM

File: stock_location_route_description/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Stock Location Routes",
```
Comment (JordiMForgeFlow) [inline stock_location_route_description/__manifest__.py]:
```suggestion
    "name": "Stock Location Route Description",
```

File: stock_location_route_description/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (JordiMForgeFlow) [inline stock_location_route_description/__manifest__.py:4]:
```suggestion
# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
{
```

File: stock_location_route_description/models/__init__.py (hunk 2 above)
Comment (JordiMForgeFlow) [reply stock_location_route_description/models/__init__.py:1]: @DavidJForgeFlow in python files you add logic to models, and the model name is stock.location.route, so the python file name should reflect the name of the model you are implementing :)

File: stock_location_route_description/views/stock_location_route_views.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="stock_location_route_form_view_gubi" model="ir.ui.view">
```
Comment (JordiMForgeFlow) [inline stock_location_route_description/views/stock_location_route_views.xml]:
```suggestion
    <record id="stock_location_route_form_view" model="ir.ui.view">
```

File: stock_location_route_description/views/stock_location_route_views.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+        <field name="name">stock.location.route.form</field>
```
Comment (JordiMForgeFlow) [inline stock_location_route_description/views/stock_location_route_views.xml]:
```suggestion
        <field name="name">stock.location.route.form - stock_location_route_description</field>
```

File: stock_location_route_description/models/__init__.py (hunk 2 above)
Comment (DavidJForgeFlow) [reply stock_location_route_description/models/__init__.py:1]: I just saw that the model is named is stock_location_route.py, so that is really my bad :c

**Comment by JudithAforgeFlow** [comment]
@AaronHForgeFlow @MiquelRForgeFlow @JordiMForgeFlow

File: stock_location_route_description/README.rst [hunk 7]
```diff
@@ -0,0 +1,81 @@
+============================
+Stock Location Route Description
+============================
```
Comment (JordiMForgeFlow) [inline stock_location_route_description/README.rst]: The '=' needs to cover the entire title :)

**Review by JordiMForgeFlow — APPROVED** [review]
LGTM 👍🏼

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM :)

File: stock_location_route_description/models/stock_location_route.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class StockLocationRoute(models.Model):
+    _inherit = "stock.location.route"
+
+    description = fields.Char()
```
Comment (rousseldenis) [inline stock_location_route_description/models/stock_location_route.py]: Shouldn't it be great to have it translatable (as name is)?

File: stock_location_route_description/models/stock_location_route.py [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+    def name_get(self):
+        result = []
+        if self.env.context.get("show_description", False):
```
Comment (JordiMForgeFlow) [inline stock_location_route_description/models/stock_location_route.py:14]: what happens if there is no show_description context?

Comment (JudithAforgeFlow) [reply stock_location_route_description/models/stock_location_route.py:14]: I'm working on it, I forgot it

**Review by JordiMForgeFlow — APPROVED** [review]
Code review LGTM! Nice work @JudithAforgeFlow ! 😄

**Pull request merged**